spec/ui_spec.rb

```diff
@@ -0,0 +1,38 @@
+require './lib/board.rb'
+include UI
+
+describe UI do
+     let(:board){ Board.new }
+  
+  describe "#decide_other_sign" do
+    it "takes the sign and returns the other one" do
+      expect(decide_other_sign('X')).to eql('O')
+    end
+  end
+
+
+  describe "#play_again" do
+    it "takes an answer as Y or N and returns it" do
+      stub(:gets) {"N"}
+      expect(play_again?).to eql('N')
+    end
+  end
+
+  describe "#choose_num" do
+    it "returns the chosen cell if only its available" do
+      board.update_board(3, "X")
+      stub(:gets) {"2"}
+      expect(choose_num(board)).to eql("2")
+    end
+  end
+
+  describe "#choose_sign" do
```

Comment:
Maybe you can try testing for when Incorrect Sign value is given.

Contact me and maybe we can brainstorm together =).

For now though, this is not required

Reply:
The change for this method has been done